project1-sps/script.js

```diff
@@ -1,64 +1,54 @@
-var win = 0; 
-var draw = 0; 
-var lose = 0; 
-var numberOfGames =0; 
-
+var win = 0;
+var draw = 0;
+var lose = 0;
+var numberOfGames = 0;
 
 var main = function (input) {
-  var computerGuess = rollDice(); 
+  var computerGuess = rollDice();
 
-  if (computerGuess !=0){
-    numberOfGames +=1;
-    var winPercent = Math.floor(win/numberOfGames *100); 
-    var drawPercent = Math.floor(draw/numberOfGames *100); 
-    var LosePercent = Math.floor(lose/numberOfGames *100); 
+  if (computerGuess != 0) {
```

Comment:
would computer guess ever be 0? since rollDice() returns 1 to 3? This if function may not be working as intended